charactersheet/charactersheet/models/dm/encounter_sections/npc.js

```diff
@@ -42,20 +42,10 @@ function NPC() {
     // UI Methods
 
     self.longDescription = ko.pureComputed(function() {
-        return self._plainTextDescription().substr(0, 200).trim() + '...';
+        return Utility.markdown.asPlaintext(self.description()).substr(0, 200).trim() + '...';
     });
 
     self.shortDescription = ko.pureComputed(function() {
-        return self._plainTextDescription().substr(0, 100).trim() + '...';
-    });
-
-    // Private Methods
-
-    /**
-     * Returns the same text as the description, but without HTML/MD artifacts.
-     */
-    self._plainTextDescription = ko.pureComputed(function() {
-        var myString = self.description() || '';
-        return marked(myString).replace(/<(?:.|\n)*?>/gm, '');
+        return Utility.markdown.asPlaintext(self.description()).substr(0, 100).trim() + '...';
```

Comment:
It might be valuable to add a ```if (self.description())``` before the value is returned. This way, a blank description won't produce just three dots. 
Also, not sure if we should check if the description exceeds the 100 or 200 chars before adding the ellipses. I didn't do that for the items or magic items, but I think it might be worth it. Thoughts?